deployment/terraform/modules/ec2/main.tf

```diff
@@ -347,3 +347,21 @@ resource "aws_elb" "https" {
     unhealthy_threshold = 4
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "cpu_utilization" {
```

Comment:
`resource "aws_cloudwatch_metric_alarm" "app_server_cpu_utilization"`
We may have other CPU utilization alarms.